#!/usr/bin/env python
PACKAGE = "ssh_object_identification"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("double_minCylinderRadiusLimits", double_t, 0, "Parameter of cylinder min radius limits", 0.0, 0.0, 1.0)
gen.add("double_maxCylinderRadiusLimits", double_t, 0, "Parameter of max cylinder radius limits", 0.2, 0.0, 1.0)
gen.add("double_minSphereRadiusLimits", double_t, 0, "Parameter of min sphere radius limits", 0.0, 0.0, 1.0)
gen.add("double_maxSphereRadiusLimits", double_t, 0, "Parameter of max sphere radius limits", 0.3, 0.0, 1.0)
#gen.add("double_rectangularInnerProductThreth", double_t, 0, "Parameter of Rectangular inner products", 0.2, 0.0, 1.0)

exit(gen.generate(PACKAGE, "Config", "sshObjectIdentificationParams"))
